internal/rbd/mirror.go

```diff
@@ -33,6 +33,39 @@ func (ri *rbdImage) EnableImageMirroring(mode librbd.ImageMirrorMode) error {
 	}
 	defer image.Close()
 
+	parentInfo, err := image.GetParent()
+	if err != nil {
```

Comment:
if there is no parent for the image this will fail with ` librbd.ErrNotFound` error isn't it?